topanswers.js

```diff
@@ -1,36 +1,77 @@
 window.addEventListener('load', async () => {
-    const htmlFromString = html => {
-        const el = document.createElement('template');
-        el.innerHTML = html;
-        return el.content.childNodes;
-    };
+    const htmlFromString = (html) => {
+        const el = document.createElement('template')
+        el.innerHTML = html
+        return el.content.childNodes
+    }
+    const updateList = (data, id) => {
```

Comment:
Missing semicolon, newline between functions.
```suggestion
    };
    
    const updateList = (data, id) => {
```

Reply:
is was the Prettier config  problem i ll fix that